admin/partials/admin-monk-select-render.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Provide the view for the monk_select_render function
+ *
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+
+	$options = get_option( 'monk_language' );
+	?>
+	<select name='monk_language[language_select]'>
```

Comment:
Don't use single quotes in HTML.
Localize with `_e` function the option tag label